routines/SAMICLOG.m

```diff
@@ -96,10 +96,11 @@ d LOGIT(CLOGROOT,entry)
  ; CLOGROOT points to the log
  ;
  n logdt
- s logdt=$$VAPALSDT^SAMICASE($$NOW^XLFDT)
+ ;s logdt=$$VAPALSDT^SAMICASE($$NOW^XLFDT)
```

Comment:
Did you comment this, because you thought you might need it here later?

Reply:
For the logs on the intake form, I'm not using the standard VAPALS date, because it does not have the time in it.. so I'm asking the Kernal for a DTTM stamp that has the date as we use it elsewhere, but also has the time.